_docs/latest/setup/forseti-on-gke.md

```diff
@@ -220,6 +220,18 @@ helm upgrade -i forseti forseti-security/forseti-security \
     --set-string serverConfigContents="$(gsutil cat gs://<BUCKET_NAME>/configs/forseti_conf_server.yaml | base64 -)" \
     --values=forseti-values.yaml
 ```
+
```

Comment:
I think you only need to edit the _docs/_latest files. Changing the _docs/latest is not needed.